### PR TITLE
Handle disconnected wallet state

### LINKED ISSUE #183: handle disconnected wallet state
Ensure all pages handle the state where no wallet is connected. Designs have been updated with a new graphic.

### PR DESCRIPTION
**Pull request by mobiustripper42**
Removed matic network config.
Removed arweave/bundlr test pages

Attempted to standardize pages so the top spacing come from pages.index.tsx.

TESTING:
Connect and disconnect wallet at various app locations to make sure the disconnected message appears.
Once connected switch to unsupported network to make sure unsupported message appears.

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -7,11 +7,32 @@ import { DashboardPage } from './DashboardPage';
...
+export enum RouteKey {
+  EMBALM_PAGE,
+  DASHBOARD_PAGE,
+  DASHBOARD_DETAIL,
+  ARCHEOLOGIST_PAGE,
+  BUNDLER_PAGE,
+  TEMP_RESURRECTION_PAGE,
+}
+
+export const RoutesPathMap: { [key: number]: string } = {
```
Comment (mobiustripper42) [inline src/pages/index.tsx:25]: Created this `map` for the route paths. I'm not 100% sure of the name and would take any suggestion. Or even a different way to store those? Maybe just a helper function.

File: src/lib/config/NetworkConfigProvider.tsx [hunk 2]
```diff
@@ -1,29 +1,45 @@
...
+  const isSupportedChain = supportedChainIds.includes(networkConfig.chainId);
+
+  const supportedNetworkNames = Object.values(networkConfigs)
+    .filter(config => supportedChainIds.includes(config.chainId))
+    .map(config => config.networkShortName);
 
   return (
-    <NetworkConfigContext.Provider value={networkConfig}>{children}</NetworkConfigContext.Provider>
+    <NetworkConfigContext.Provider value={networkConfig}>
+      <SupportedNetworkContext.Provider
```
Comment (mobiustripper42) [inline src/lib/config/NetworkConfigProvider.tsx:40]: Add an additional context to return `SuppportedNetwork` properties. Attempted to add directly to the NetworkConfig, but in the end separating them made more sense.

File: src/lib/config/useSupportedNetwork.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { createContext, useContext } from 'react';
```
Comment (mobiustripper42) [inline src/lib/config/useSupportedNetwork.ts:1]: This seems like the correct pattern for filename and function names. But maybe something else makes more sense.

File: src/components/Navbar.tsx [hunk 4]
```diff
@@ -2,7 +2,7 @@ import { Flex, Image } from '@chakra-ui/react';
 import React from 'react';
 import { useNavigate } from 'react-router-dom';
 
-export function DevNavbar({ children }: { children: React.ReactNode }) {
+export function Navbar({ children }: { children: React.ReactNode }) {
```
Comment (knoll3) [inline src/components/Navbar.tsx:5]: Oops, forgot about this.

File: src/components/Navbar.tsx [hunk 5]
```diff
@@ -15,12 +15,14 @@ export function DevNavbar({ children }: { children: React.ReactNode }) {
       backgroundColor="brand.50"
       alignItems="center"
       px={6}
+      boxShadow="0px 1px 0px #29262F"
```
Comment (knoll3) [inline src/components/Navbar.tsx]: Can we add this to the color palette under `theme` so it's easier to modify later?

**Review by knoll3 — APPROVED** [review]

File: src/pages/index.tsx (hunk 1 above)
Comment (knoll3) [reply src/pages/index.tsx:25]: I think this is the logical place for this. It's an improvement but I'm afraid people will forget about it and just use the old way. Creating a custom `navigate` function that only accepts the `RouteKey` might help people use it properly, but they still might use the old function.

File: src/lib/config/useSupportedNetwork.ts (hunk 3 above)
Comment (knoll3) [reply src/lib/config/useSupportedNetwork.ts:1]: Seems good to me.

File: src/components/Navbar.tsx (hunk 5 above)
Comment (knoll3) [reply src/components/Navbar.tsx]: I added it while resolving the merge conflicts.

**Pull request merged**